veracruz-server-sgx/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
```

Comment:
This file needs the copyright/licensing header at the top that all other `Cargo.toml` files have.